R/learner_abess_classif_abess.R

```diff
@@ -0,0 +1,155 @@
+#' @title Classification Adaptive Best-subset Selection Learner
+#' @author Jin Zhu
```

Comment:
Author should be your GitHub handle not your name (@rlwanggg)

Reply:
thank u